torch/onnx/utils.py

```diff
@@ -304,7 +304,9 @@ def _model_to_graph(model, args, verbose=False, training=False,
     if isinstance(model, torch.jit.ScriptModule):
         assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule"
         try:
-            method_graph, params = torch._C._jit_pass_lower_graph(model.forward.graph, model._c)
+            # Running the model once to ensure the graph is profiled.
+            model(*args)
+            method_graph, params = torch._C._jit_pass_lower_graph(model.forward._profiled_graph, model._c)
```

Comment:
I am trying to understand this. So we don't have to enable the profiler anywhere (by setting any flag or calling any API) to indicate that we need the `model.forward._profiled_graph` populated - it is just done by default the first time the model is run, is this correct? I am asking because I don't see any place where we enabled any flag or such for profiler (or maybe I missed it).

Reply:
Yes, by default the profiling mode is turned on. AFAIK there is no user level api to turn off profiling mode, that's why I didn't add code to enable the mode here.